opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -82,6 +82,18 @@ message Profile {
   repeated Location location = 4;
   // Array of locations referenced by samples.
   repeated int64 location_indices = 15;
+  // Whether addresses within locations have been normalized to be
```

Comment:
I'm not convinced from this description that this new field is necessary, to be honest. But I wasn't in the Profiling WG meeting where this was discussed so I probably miss some nuances. It would be good to mention in more detail what functionality or knowledge is missed if this field is not added.

In general, translating a runtime address for a profile into objdump-compatible address goes as `profile_address - rtMappingVaddr + peSegmentVaddr + rtMappingFileOff - peSegmentFileOff`. If a particular profile translation step wants to actually rewrite addresses in the profile into their objdump-compatible versions, then I think the same step should be able to set `Mapping.memory_start`, `Mapping.memory_limit` and `Mapping.file_offset` fields into their values from the PE header so that the above formula in downstream processing yields zero offset.